ui/src/app/base/components/Switch/Switch.tsx

```diff
@@ -0,0 +1,29 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+
+type Props = {
+  className?: string;
+} & React.PropsWithoutRef<JSX.IntrinsicElements["input"]>;
```

Comment:
For some reason that won't resolve the types correctly for me. I think it's because this is a checkbox which has a read-only `value` whereas the React type has it as writeable.

Reply:
@squidsoup any thoughts?